src/utils/explorers.js

```diff
@@ -0,0 +1,23 @@
+import { SUPPORTED_NETWORKS, isMainNet } from '../utils/networkUtils';
+
+const getEtherScanBaseURL = networkId => {
+	const network = SUPPORTED_NETWORKS[networkId];
+
+	if (isMainNet(networkId) || network == null) {
+		return 'https://etherscan.io';
+	}
+
+	return `https://${network.toLowerCase()}.etherscan.io`;
+};
+
+export const getEtherscanTxLink = (networkId, txId) => {
```

Comment:
might be worthwhile supporting both network by name and network by ID. You could then make this a standalone module that we can pull into https://github.com/Synthetixio/synthetix as well

Reply:
Sure, will add support for it. Can we first add it here, and then I'll export it into a module?